docs/workflow.rst

```diff
@@ -0,0 +1,409 @@
+.. _workflow-page:
+
+*********
+Workflows
+*********
+
+.. note::
```

Comment:
As with modules, my preference would be to drop this down below the intro sentence.